### ISSUE TITLE
New plugin: stainedglass

### ISSUE THREAD

**Issue by manuelsongokuh** [olear/openfx-arena#68]
hello

i find good effect for filter and not distort:
http://www.fmwconcepts.com/imagemagick/stainedglass/index.php

this is possibile for future roadmap with keep calm ok?

**Comment by olear** [comment]
See issue #33 :)

**Comment by manuelsongokuh** [comment]
yes, i did see, but i open issue for new filter name staindglass or other name, this mean like as new filter ok?

if you want to use #33 will be long page if i will add new filter?

**Comment by olear** [comment]
Ok :)

Will start to add new effects soon, just need to wait on ImageMagick for bugfix then release openfx-arena 0.7. And start to add new effects in v0.8.

**Comment by manuelsongokuh** [comment]
ok i'm agree you, i dont say for 0.7 but future or roadmap like as next feature ok? i'm prepare new issue for new filter a moment..

i'm fine if you will add feature for 0.8 but can move to 0.9 it's not bad ok?
important i write new issue for memo note ok?

**Comment by olear** [comment]
This should be possible with the new GMIC plugin(s).

**Comment by manuelsongokuh** [comment]
ok where i can try gmic plugin?
thank you

**Comment by olear** [comment]
Not available yet.

**Comment by manuelsongokuh** [comment]
ok, there is a link for plugin gmic? or roadmap o news? thank you

**Comment by olear** [comment]
Should be available in Natron 2.1, or 2.2. That's all I can say at this moment.
